var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -38,6 +41,7 @@ class PyMaestrowf(PythonPackage):
 
     depends_on('python@2.7:2.8,3.5:',  type=('build', 'run'))
```

Comment:
I was considering that, but I'm not sure if users are still using this in a 2.7 stack. If there's a way to do it I was thinking allowing <3.6 for older versions and requiring 3.6+ for newer versions would be best.

Reply:
Yes, the correct syntax is:
```python
depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
depends_on('python@3.6:', when='@1.1.9:', type=('build', 'run'))
```
assuming that 1.1.9 was the first version that requires Python 3.6+

Reply:
I'll go through the tags in Maestro to make sure that's right; but iirc, 1.1.9 was the one that made the min 3.6 (it's been a while since I've done a release)